test/Microsoft.AspNetCore.Server.KestrelTests/EngineTests.cs

```diff
@@ -481,8 +490,8 @@ public void ConnectionCanReadAndWrite(TestServiceContext testContext)
 
                 using (var connection = server.CreateConnection())
                 {
-                    await connection.SendEnd(
-                        "GET / HTTP/1.0",
+                    await connection.Send(
+                        "POST / HTTP/1.0",
```

Comment:
I had added a body to the message and for correctness sake changed the method to POST. Then I removed the body and forgot to change the method back to GET. Reverting.